ui/.eslintrc.json

```diff
@@ -30,15 +30,8 @@
         "plugin:@angular-eslint/template/process-inline-templates"
       ],
       "rules": {
+        "curly": "error",
         "unused-imports/no-unused-imports": "error",
-        "@stylistic/semi": [
-          "error",
-          "always"
-        ],
-        "@stylistic/quote-props": [
-          "warn",
-          "consistent"
-        ],
```

Comment:
could you fix formatting, why did you change the order around?

Reply:
to clear rule cateogry. I commented on this commit.
https://github.com/OpenEMS/openems/pull/2676/commits/4a1365c43ba1826b06f235a300fb739284304ffe